src/lib.rs

```diff
@@ -518,6 +520,130 @@ fn collect_law_sections(
         section_numbers: law_sections,
     }
 }
+pub fn get_search_page(cli: &Cli) -> (bool, Url, String) {
```

Comment:
Is this function grouped with associated functions? In general, does the order of the functions in the file make some sort of sense?